Packages/doc/wavebuilder.rst

```diff
@@ -0,0 +1,28 @@
+.. _wavebuilder:
+
+WaveBuilder
+============
+
+The WaveBuilder is used to create a series of related sweeps called a stimulus
+set. A sweep is composed of up to 99 epochs. Each epoch can contain one of
+seven different waveforms types, arbitrary combinations of waveform types, or a
+custom waveform. A custom waveform can be a sweep from a different stimulus
+set, or a waveform generated by the user using other tools. Custom code to
+automate analysis or data acquisition, executed during acquisition, can be
+associated with a stimulus set as well.
```

Comment:
The WaveBuilder is used to create a series of related sweeps called a stimulus set. A sweep is composed of up to 99 epochs. Each epoch can contain one of seven different waveform types, arbitrary combinations of waveform types, or a custom waveform. A custom waveform can be a sweep from a different stimulus set, or a waveform generated by the user using other tools. Custom code to automate analysis or data acquisition may be linked with a stimulus set using the WaveBuilder.